packages/common-server/src/DConfig.ts

```diff
@@ -31,73 +31,38 @@ export enum LocalConfigScope {
 
 let _dendronConfig: DendronConfig | undefined;
 
+/**
```

Comment:
There are still a few methods in `DConfig` that needs to be moved elsewhere before I can completely remove it, and the earliest we can do that is when we remove the dev flag for engine v3.